docusaurus.config.js

```diff
@@ -160,7 +160,7 @@ module.exports = {
       copyright: `Copyright © ${new Date().getFullYear()} OpenJS Foundation and Electron contributors.`,
     },
     algolia: {
-      apiKey: 'c9e8f898b3b32afe40f0a96637e7ea85',
+      apiKey: 'fdc2cf6080e499639d7e6b0278851ed4',
```

Comment:
You should also add the `appId` we provided in the email.

Reply:
So it is either

index name: `electronjs`
```suggestion
      appId: 'MG3SRMK3K0',
      apiKey: 'c9e8f898b3b32afe40f0a96637e7ea85',
```

or

index name: `beta-electronjs`
```suggestion
      appId: '4PUIL4O0HL',
      apiKey: '681405e5bb1f0ab10cff4f14b94217b7',
```

Reply:
For the first one, would it be `fdc2cf6080e499639d7e6b0278851ed4` for the `apiKey`?

Reply:
Indeed you are right sorry for the confusion